.changeset/five-moons-complain.md

```diff
@@ -0,0 +1,6 @@
+---
+"@directus/api": patch
+"@directus/env": patch
```

Comment:
Should these be split up into separate change sets?

Reply:
I dont think thats needed in this case 😄